guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -2,13 +2,12 @@
 
 = Configuring Repositories
 
-ifeval::["{build}" == "foreman"]
-This procedure is only for Katello plug-in and {RHEL}-based operating system users.
-endif::[]
-
 Use this procedure to enable the repositories that are required to install {ProductName}.
 
 .Procedure
+ifeval::["{build}" == "foreman"]
+Skip steps involving `subscription-manager` when not installing on {RHEL} operating system.
```

Comment:
```suggestion
This procedure is only for Katello plug-in users. Skip steps involving `subscription-manager` when not installing on the {RHEL} operating system.
```

Reply:
Return the Katello bit back and add an article for "operating system".

Reply:
> Return the Katello bit back and add an article for "operating system".

I am not following. What article? Sorry :-)